code-challenges.test.js

```diff
@@ -65,5 +153,42 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("firstVowelFinder",() => {
+    it("returns the index of the first vowel",() => {
+      //Function: firstVowelFinder, Variable name: vowelTester1, expected output
+      expect(firstVowelFinder(vowelTester1)).toEqual(1)
+      //Function: firstVowelFinder, Variable name: vowelTester2, expected output
+      expect(firstVowelFinder(vowelTester2)).toEqual(0)
+      //Function: firstVowelFinder, Variable name: vowelTester3 expected output
+      expect(firstVowelFinder(vowelTester3)).toEqual(2)
+      })
+  })
+//Yarn jest fail received: 
+//firstVowelFinder › returns the index of the first vowel
+//ReferenceError: firstVowelFinder is not defined
+
 
 // b) Create the function that makes the test pass.
+
+// My first code looked like this, which was incorrect: 
+//const firstVowelFinder = (vowelTester1, vowelTester2, vowelTester3) => {
+//     return firstVowelFinder.indexOf('a','e','i','o','u')
+// }
+
+//*** After searching online to gain a better sense of direction after encountering difficulty, I found the following video on YouTube which I used to refine my code: 
+//https://www.youtube.com/watch?v=Xxgauhcq_y8 
+// I then read up on the .search() method on W3Schools to understand how this method is used. I learned that the search() method returns the index of the first match, which in the case of this code would be the first vowel that matches. I have tested this function against different variables and have received the expected output each time. Additionally, I noticed in the YouTube video that the creator used the "g" modifier in his code. I wasn't familiar with this, so I also used W3Schools to learn more. The "g" modifier specifies a global match, which finds all matches as opposed to the first match only (from what I understand). I was able to remove the "g" modifier from my code without a change to its functionality. 
+//https://www.w3schools.com/jsref/jsref_search.asp#:~:text=The%20search()%20method%20returns,()%20method%20is%20case%20sensitive 
+
+
+//Function takes in a string and logs the index of the first vowel. 
+//Function name: firstVowelFinder
+//Input: string
+const firstVowelFinder = (string) => {
+// Use .search() method to return the position of the first vowel in the string
+return string.search(/[a,e,i,o,u,A,E,I,O,U]/)
+
+}
```

Comment:
Excellent approach to this problem!  Super excited that you were using resources to explore different approaches.  I would definitely recommend researching regular expressions (or regex for short).  The syntax for reg ex is those slashes you used, and commonly using the global modifier.  They are very handy, but would recommend researching it.  Here is an article that has a lot of info on it: https://blog.bitsrc.io/a-beginners-guide-to-regular-expressions-regex-in-javascript-9c58feb27eb4